Book-A-Desk.Infrastructure/DynamoDbEventStore.fs

```diff
@@ -19,7 +19,7 @@ module rec DynamoDbEventStore =
                 .Create<ReservationEvent>(
                     dynamoDbClient,
                     tableName = "ReservationEvents",
-                    createIfNotExists = false)
```

Comment:
I think you can undo this change. We did this to make it run locally for you, but it's not something that we want in the deployed code

Reply:
But doesn't that mean that everyone who tries to run it locally (or just running it for the first time in general) has to create the database themself?

Reply:
Yes, but it's supposed to be done already in the setup part of the README. The important point is that we do not want the production code to create tables accidentally.